### ISSUE TITLE
Does it support GPU acceleration?

### ISSUE THREAD

**Issue by Mateus13000** [FerryAr/lhroot#33]
I just wanna know if it's possible to have access to the GPU acceleration and how...

**Comment by YourSandwich** [comment]
Yeah, checkout my Docu for this https://wiki.archworks.co/en/Andorid/Linux-with-3D-Acceleration

It is based on https://github.com/cheadrian/termux-chroot-proot-wine-box86_64/tree/main

**Comment by Mateus13000** [comment]
> Yeah, checkout my Docu for this https://wiki.archworks.co/en/Andorid/Linux-with-3D-Acceleration
> 
> It is based on https://github.com/cheadrian/termux-chroot-proot-wine-box86_64/tree/main

A little question, why is my lhroot killed when something too heavy is running?

**Comment by ihtarlik** [comment]
That's probably the phantom process killer in Android 12+, not an lhroot issue per se.
See:
https://kskroyal.com/disable-phantom-process-killer-in-android-12-13/

**Comment by Mateus13000** [comment]
> That's probably the phantom process killer in Android 12+, not an lhroot issue per se. See: https://kskroyal.com/disable-phantom-process-killer-in-android-12-13/

But I'm using Android 10...

**Comment by viridivn** [comment]
> > That's probably the phantom process killer in Android 12+, not an lhroot issue per se. See: https://kskroyal.com/disable-phantom-process-killer-in-android-12-13/
> 
> But I'm using Android 10...

Could be the OOM killer

**Comment by MateusL13** [comment]
> > > That's probably the phantom process killer in Android 12+, not an lhroot issue per se. See: https://kskroyal.com/disable-phantom-process-killer-in-android-12-13/
> > 
> > 
> > But I'm using Android 10...
> 
> Could be the OOM killer

Is there any way to disable it?

**Comment by viridivn** [comment]
OOM killer means you ran out of memory. If that's the case you can either get a phone with more RAM or find a way to make the process use less RAM

**Comment by MateusL13** [comment]
> Yeah, checkout my Docu for this https://wiki.archworks.co/en/Andorid/Linux-with-3D-Acceleration
> 
> It is based on https://github.com/cheadrian/termux-chroot-proot-wine-box86_64/tree/main

Can you please make another guide for GPU acceleration?? 
This one doesn't work for me unfortunately, I would like something for Ubuntu with no desktop, for running media servers like jellyfin, that could use the GPU for decoding/encoding 😁

**Comment by YourSandwich** [comment]
@MateusL13 It does not work for me anymore since Android version 13. Didn't have time to solve the issues.

**Comment by MateusL13** [comment]
> [@MateusL13](https://github.com/MateusL13) It does not work for me anymore since Android version 13. Didn't have time to solve the issues.

That's totally ok, the termux guide that you sent has exactly what I want! Thank you, and sorry for bothering 😅

**Comment by YourSandwich** [comment]
@MateusL13 
Your Jellyfin GPU acceleration probably won't work this way, this method uses Mesa Zink driver for the 3D acceleration, which probably does not support VAAPI or any of the Jellyfin hardware acceleration methods.

You may have better luck with a real Linux OS or if you manage to run Docker in Termux.
Most Kernels don't have the required docker modules.

Otherwise, Android 16 with its inbuilt Linux support may be a better bet.
